onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -153,7 +157,7 @@
                     android:name="android.support.PARENT_ACTIVITY"
                     android:value="org.onebusaway.android.ui.HomeActivity"/>
         </activity>
-        <activity android:name="org.onebusaway.android.ui.ReportTripProblemActivity"
+        <activity android:name=".report.ui.ReportTripProblemActivity"
```

Comment:
Let's keep the full package name here, just for consistency with the other activities.